client/src/Security/LoginFormAuthenticator.php

```diff
@@ -57,8 +58,6 @@ public function authenticate(Request $request): Passport
                 } catch (AuthenticationException $e) {
                     throw $e;
```

Comment:
I know this isn't your code, but this is an odd pattern if we aren't doing anything before just re-throwing the same exception. Would expect logging or something.

Reply:
I had assumed that the logging was done elsewhere, part of the brute force protection.

Reply:
The exception is caught by the `onAuthenticationFailure()` method further down which then decides what to do with it. In this case, it sets the error and redirects to the login page (the login page will check the exception and show the appropriate error message)